torchdynamo/utils.py

```diff
@@ -65,10 +64,59 @@
 }
 
 
-@functools.lru_cache(None)
 def init_logging():
     if "PYTEST_CURRENT_TEST" not in os.environ:
         logging.config.dictConfig(LOGGING_CONFIG)
+        logger = logging.getLogger(config.log_name)
+        logger.setLevel(config.log_level)
+
+
+def get_logger():
```

Comment:
What's the benefit of wrapping the logging functions like this? At least within `pytorch/pytorch` I've mostly seen the syntax for the python `logging` module used directly

Reply:
I wanted to get the log name from the config, instead of it being module-specific.

Reply:
I think it is better to have per-file usage of logging.  At the top of every file say:
```
log = logging.getLogger(__name__)
```
Then call `log.info()`, `log.warning()`, etc.

The big advantage of doing it this way is you can control log levels at a finer granularity.  E.g. you can do:
```
logging.getLogger("torchdynamo.symbolic_convert").setLevel(logging.ERROR)
```
to silence logs from a specific file.

This also lets us point very noisy things (like bytecode dumps) in their own logging stream.